.flowconfig

```diff
@@ -11,7 +11,7 @@ module.file_ext=.less
 # stub out less imports.
 module.name_mapper.extension='less' -> 'empty/object'
 module.name_mapper='@khanacademy/\(perseus[^/]*\)' -> '<PROJECT_ROOT>/packages/\1/src/index.js'
-module.name_mapper='@khanacademy/kas' -> '<PROJECT_ROOT>/packages/kas/kas.js'
+module.name_mapper='@khanacademy/kas' -> '<PROJECT_ROOT>/packages/kas/src/index.js'
```

Comment:
I wonder if we just merge this with line 13 (and remove `perseus` from line 13).

Reply:
I think it'll be easier to optimize .flowconfig once the dust settles.  `git` is better at resolving conflicts when there isn't multiple changes to the same line in a file.

Reply:
FTR, we can't change `@khanacademy/\(perseus[^/]*\)` to `@khanacademy/\([^\]*\)` because that would match imports of wonder-blocks and wonder-stuff too.